agora-test/Spec/Sample/Shared.hs

```diff
@@ -131,3 +147,42 @@ defaultProposalThresholds =
     , create = Tagged 1
     , startVoting = Tagged 10
     }
+
+------------------------------------------------------------------
+
+treasuryOut :: TxOut
+treasuryOut =
+  TxOut
+    { txOutAddress = Address trCredential Nothing
+    , txOutValue = minAda
+    , txOutDatumHash = Nothing
+    }
+
+{- | Arbitrary 'CurrencySymbol', representing the 'CurrencySymbol'
+     of a valid governance authority token (GAT).
+-}
+gatCs :: CurrencySymbol
+gatCs = "73475cb40a568e8da8a045ced110137e159f890ac4da883b6b17dc651b3a8049"
+
+trValidator :: Validator
+trValidator = mkValidator (treasuryValidator gatCs)
+
+-- | `ScriptCredential` used for the dummy treasury validator.
+trCredential :: Credential
+trCredential = ScriptCredential $ validatorHash trValidator
+
+-- | `TokenName` for GAT generated from address of `mockTrEffect`.
+gatTn :: TokenName
+gatTn = validatorHashToTokenName $ validatorHash mockTrEffect
+
+-- | Mock treasury effect script, used for testing.
+mockTrEffect :: Validator
+mockTrEffect = mkValidator $ noOpValidator gatCs
+
+------------------------------------------------------------------
+
+minAda :: Value
+minAda = Value.singleton "" "" 10_000_000
+
+withMinAda :: Value -> Value
```

Comment:
You reckon? It came from [this suggestion](https://github.com/Liqwid-Labs/agora/pull/37#discussion_r866501360) @chfanghr

Reply:
I actually used it everywhere in the sample script contexts.

Reply:
So, this is obviously just an opinion thing. But the reason I say it is because of the more familiar ways of applying it directly:
```haskell
withMinAda $
  mconcat
    [ foo
    , bar
    ]
```
vs
```haskell
mconcat
  [ foo
  , bar
  , minAda
  ]
```
I think this reads much more clearly, and makes it obvious that there's no magic involved in adding the minAda, it's just a static value. Technically speaking, min ADA isn't just a static amount of ADA. It is calculated through a relatively intricate formula that is calculated per UTXO, and doing it this way (`minAda` directly) makes it clear that this is a simplification.

Reply:
Ah, I see your point. I've changed it to:

```haskell
mconcat
  [ Value.singleton gatCs gatTn 1
  , minAda
  ]
```

lgtm?